src/containers/FormikForm/formikDraftHooks.ts

```diff
@@ -79,7 +79,11 @@ export function useFetchArticleData(articleId: string | undefined, locale: Local
     if (!updatedArticle.id) throw new Error('Article without id gotten when updating status');
 
     const statusChangedDraft = await draftApi.updateStatusDraft(updatedArticle.id, newStatus);
-    const updated = await transformArticleFromApiVersion(statusChangedDraft, locale);
+    const updated = await draftApi
+      .fetchDraft(updatedArticle.id, locale)
+      .then(art =>
+        transformArticleFromApiVersion({ ...art, status: statusChangedDraft.status }, locale),
+      );
```

Comment:
Det blir litt uryddig å blande await og .then(). Tenker det er bedre å bare bruke await som ellers i fila. Noe sånt:
```suggestion
    const article = await draftApi.fetchDraft(updatedArticle.id, locale);
    const updated = await transformArticleFromApiVersion({ ...article , status: statusChangedDraft.status }, locale);
```